src/main/telemetry/telemetry.c

```diff
@@ -69,6 +70,9 @@ void telemetryInit(void)
     initSmartPortTelemetry();
     initLtmTelemetry();
     initMAVLinkTelemetry();
+    #ifdef TELEMETRY_IBUS
```

Comment:
Indenting is inconsistent with rest of (or larger part of) code here - `#directive` should be in leftmost column / use `# directive` to nest `#if`s etc ..

Conditional code is normally indented:
     initMAVLinkTelemetry();
 +    #ifdef TELEMETRY_IBUS
  
            
  Write  Preview

Indenting is wrong here - `#directive` should be in leftmost column / use `# directive` to nest `#if`s etc ..

Conditional code is normally indented:
```
    initMAVLinkTelemetry();
#ifdef TELEMETRY_IBUS
    initIbusTelemetry();
#endif
    telemetryCheckState();